unlock-app/src/__tests__/components/lock/IframeService.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import * as rtl from 'react-testing-library'
+import { lockPage, unlockPage } from '../../../services/iframeService'
+
+import IframeService from '../../../components/lock/IframeService'
+
+jest.mock('../../../services/iframeService.js')
+
+describe('IframeService', () => {
+  describe('if there is a valid key', () => {
+    it('should call the iframeService unlockPage', () => {
```

Comment:
Let's add tests to make sure that `modalShown` is also taken into account please!

Reply:
added a new test, I'm going to leave this here for you and Chris to monkey around with the name